### PR TITLE
Adds RELEASING.md

### PR DESCRIPTION
**Pull request by oldergod**

### CODE REVIEW

**Review by dnkoutso — DISMISSED** [review]
LGTM with one comment

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Releasing
+=========
+
+### Prerequisite: Sonatype (Maven Central) Account
+
+Create an account on the [Sonatype issues site][sonatype_issues]. Ask an existing publisher to open
+an issue requesting publishing permissions for `com.squareup` projects.
```
[the remark this answers is not in the document]
Comment (oldergod) [reply RELEASING.md]: fixed

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    ```
+    sed -i "" \
+      "s/VERSION_NAME=.*/VERSION_NAME=$NEXT_VERSION/g" \
+      `find . -name "gradle.properties"`
+    git commit -am "Prepare next development version."
+    git push
+    ```
+
+7. CI will release the artifact and publish the documentation.
```
[the remark this answers is not in the document]
Comment (oldergod) [reply RELEASING.md:53]: we'll give it a try and see how it goes!

[the remark this answers is not in the document]
Comment (kdubb) [reply RELEASING.md:53]: Yes this looks correct. Note that the release workflow is keyed to the following regex:
```
on:
  push:
    tags: [ "[0-9]+.[0-9]+.[0-9]+**" ]
```

**Review by kdubb — APPROVED** [review]

**Pull request merged**